### PR TITLE
Set path intro "H2XINTRO.SMK" on cross-platform manner

### PR DESCRIPTION
**Pull request by undef21**

### CODE REVIEW

File: src/fheroes2/game/fheroes2.cpp [hunk 1]
```diff
@@ -186,7 +186,7 @@ int main( int argc, char ** argv )
             // goto main menu
             int rs = ( test ? Game::TESTING : Game::MAINMENU );
 
-            Video::ShowVideo( "heroes2/anim/H2XINTRO.smk", false );
+            Video::ShowVideo( Settings::GetLastFile( "data", "H2XINTRO.SMK" ), false );
```
Comment (ihhub) [inline src/fheroes2/game/fheroes2.cpp:189]: Hi @undef21 , why do we need this change? Which version of Homm2 are you using?

Comment (undef21) [reply src/fheroes2/game/fheroes2.cpp:189]: current path is relative from fheroes2 executable. But linux version fheroes2 contains data and config files in /home/user/.fheroes2/

Comment (ihhub) [reply src/fheroes2/game/fheroes2.cpp:189]: Data directory of GOG version doesn't contain this file.

Comment (undef21) [reply src/fheroes2/game/fheroes2.cpp:189]: Yes. This file in heroes2/anim. Should it add this directory? When i build project with option -DCONFIGURE_FHEROES2_DATA=/usr/share/fheroes2-data, game searches data files from this path and user home directory. But intro is searched  only by hardcoded path

Comment (undef21) [reply src/fheroes2/game/fheroes2.cpp]: Hi, @ihhub. I change path "heroes2/anim" on cross-platform manner

**Comment by ihhub** [comment]
Hi @undef21 , thank you very much for your contribution to the project!

**Pull request merged**